dashboard/config/foorm/forms/surveys/pd/ayw_workshop_post_survey.0.json

```diff
@@ -23,20 +23,32 @@
       "library_name":"surveys/pd/ayw_satisfaction_panel",
       "library_version": 0,
       "name":"workshop_satisfaction_panel"
-    },
+     }
+    ]
+   },
+   {
+    "name":"virtual_page",
+    "title":"Virtual Workshop Satisfaction",
+    "elements":[
     {
-      "type":"library_item",
-      "library_name":"surveys/pd/ayw_virtual_panel",
-      "library_version": 0,
-      "name":"ayw_virtual_panel"
-    }
-
+     "type": "panel",
```

Comment:
we still need to remove the panel here and move the visibleIf up to the page

Reply:
dangit...